xml/docker_building_images.xml

```diff
@@ -152,58 +160,58 @@ xml:id="docker.building.images" xml:lang="en">
 
   <para>
    Now you can create a custom &docker; image by using a
-   <literal>Dockerfile</literal>. If you want to create a custom &slea;&nbsp;12
+   <literal>Dockerfile</literal>. If you want to create a custom &slea;&nbsp;11 SP4
    image, please refer to
-   <xref linkend="sec.docker.sle_images.customizing_the_images.sles12"/>. If
-   you want to create a custom &slea;&nbsp;11 SP4 &docker; image, please refer to
-   <xref linkend="sec.docker.sle_images.customizing_the_images.sles11sp4"/>. In
+   <xref linkend="sec.docker.sle_images.customizing_the_images.sles11sp4"/>. If
+   you want to create a custom &slea;&nbsp;12 &docker; image, please refer to
+   <xref linkend="sec.docker.sle_images.customizing_the_images.sles12"/>. In
    case you would like to move your application to a &docker; container, please
    refer to <xref linkend="docker.dockerizing.application"/>.
   </para>
 
```

Comment:
Again, this is very hard to read, because it is floting text. Make it a itemized list to make it easier for the eye to distinguish between alternatives.

Reply:
fixed, see 3549c64